src/widgets/dialogs/settings/fileassociationpage.cpp

```diff
@@ -62,7 +62,7 @@ bool FileAssociationPage::saveInternal()
         if (name.isEmpty()) {
             continue;
         }
-        auto suffixes = lineEdit->text().split(c_suffixSeparator, Qt::SkipEmptyParts);
+        auto suffixes = lineEdit->text().split(c_suffixSeparator, QString::SplitBehavior::SkipEmptyParts);
```

Comment:
I think we need a qt version check macro here? vnote now has upgraded qt version to 5.15.

Reply:
https://doc.qt.io/qt-5/qstring-obsolete.html